cmake/roboschool.cmake

```diff
@@ -14,26 +14,28 @@ endfunction(append_libraries)
 ExternalProject_Add(assimp
   GIT_REPOSITORY "https://github.com/yu239/assimp"
   GIT_TAG "master"
+  INSTALL_DIR "${CMAKE_CURRENT_BINARY_DIR}/assimp"
   LOG_DOWNLOAD ON
   CMAKE_ARGS
-    -DCMAKE_INSTALL_PREFIX=<SOURCE_DIR>
+    -DCMAKE_INSTALL_PREFIX=<INSTALL_DIR>
 )
 
-ExternalProject_Get_Property(assimp SOURCE_DIR)
-set(ASSIMP_INCLUDE_PATH "${SOURCE_DIR}/include")
-set(ASSIMP_LIBRARIES "${SOURCE_DIR}/lib/libassimp.so.4")
+ExternalProject_Get_Property(assimp INSTALL_DIR)
+set(ASSIMP_INCLUDE_PATH "${INSTALL_DIR}/include")
+set(ASSIMP_LIBRARIES "${INSTALL_DIR}/lib/libassimp.so.4")
 
```

Comment:
Why this has anything to do with the error below?

[ 87%] Performing configure step for 'assimp'
-- Shared libraries enabled
-- Looking for ZLIB...
-- Checking for module 'zzip-zlib-config'
--   No package 'zzip-zlib-config' found
-- Found ZLIB: optimized;/usr/lib/x86_64-linux-gnu/libz.so;debug;/usr/lib/x86_64-linux-gnu/libz.so
-- Checking for module 'minizip'
--   No package 'minizip' found
-- Enabled formats: AMF 3DS AC ASE ASSBIN ASSXML B3D BVH COLLADA DXF CSM HMP IRRMESH IRR LWO LWS MD2 MD3 MD5 MDC MDL NFF NDO OFF OBJ OGRE OPENGEX PLY MS3D COB BLEND IFC XGL FBX Q3D Q3BSP RAW SIB SMD STL TERRAGEN 3D X X3D GLTF 3MF MMD
-- Disabled formats:
-- Could NOT find IL (missing:  IL_LIBRARIES IL_INCLUDE_DIR) 
CMake Warning at CMakeLists.txt:427 (MESSAGE):
  Build of assimp_qt_viewer is disabled.  Unsatisfied dendencies: Qt5 DevIL

Reply:
This change is just to make the installation locations of these external projects consistent. I could not reproduce the error above. I'll investigate more.

Reply:
This is not an error. Assimp gives this warning if it cannot find Qt5 DevIL.